apps/editors/tests/test_views.py

```diff
@@ -2908,3 +2908,111 @@ def test_editors_abuse_report_page(self):
+        limited_user.save()
+
+        permissions = [
+            {
+                'name': 'Add-on Reviewers',
+                'rules': 'Addons:Review',
+            },
+            {
+                'name': 'Limited Reviewers',
+                'rules': 'Addons:DelayedReviews',
+            },
+        ]
+        for perm in permissions:
+            group = Group.objects.create(name=perm['name'],
+                                         rules=perm['rules'])
+            GroupUser.objects.create(group=group, user=limited_user)
+
+    def login_as_limited_reviewer(self):
+        self.client.logout()
+        assert self.client.login(username='limited@mozilla.com',
+                                 password='password')
+
+
+class TestLimitedReviewerQueue(QueueTest, LimitedReviewerBase):
+
+    def setUp(self):
+        super(TestLimitedReviewerQueue, self).setUp()
+        self.expected_names = ['Nominated old']
+        self.url = reverse('editors.queue_nominated')
+
+        self.create_limited_user()
+        self.login_as_limited_reviewer()
+
+    def generate_files(self, subset=[]):
+        files = SortedDict([
+            ('Nominated new', {
+                'version_str': '0.1',
+                'addon_status': amo.STATUS_NOMINATED,
+                'file_status': amo.STATUS_UNREVIEWED,
+                'nomination': datetime.now()
+            }),
+            ('Nominated old', {
+                'version_str': '0.1',
+                'addon_status': amo.STATUS_NOMINATED,
+                'file_status': amo.STATUS_UNREVIEWED,
+                'nomination': datetime.now() - timedelta(days=1)
+            }),
+        ])
+        results = {}
+        for name, attrs in files.iteritems():
+            if not subset or name in subset:
+                results[name] = self.addon_file(name, **attrs)
+        return results
+
+    def test_results(self):
+        #import ipdb; ipdb.set_trace()
+        # `generate_files` happens within this test.
+        self._test_results()
+
+    def test_queue_count(self):
+        #import ipdb; ipdb.set_trace()
+        # `generate_files` happens within this test.
+        self._test_queue_count(1, 'Full Review', 1)
+
+    def test_get_queue(self):
+        #import ipdb; ipdb.set_trace()
+        # `generate_files` happens within this test.
+        self._test_get_queue()
+
+
+class TestLimitedReviewerReview(ReviewBase, LimitedReviewerBase):
+
+    def setUp(self):
+        super(TestLimitedReviewerReview, self).setUp()
+
+        self.create_limited_user()
+        self.login_as_limited_reviewer()
+
+    def test_new_addon_review_action_as_limited_editor(self):
+        self.addon.update(status=amo.STATUS_NOMINATED)
+        self.version.update(nomination=datetime.now())
+        self.version.files.update(status=amo.STATUS_UNREVIEWED)
+        response = self.client.post(self.url, self.get_dict(action='public'))
+        eq_(response.status_code, 200)  # Form error.
```

Comment:
we do `assert response.status_code == 200` now to make use of py.test's introspection features.